.editorconfig

```diff
@@ -195,5 +501,11 @@ end_of_line = crlf
 # IDE0040: Add accessibility modifiers
 dotnet_diagnostic.IDE0040.severity = warning
 
+# Rules for generated code for template engine
+[{*/obj/*/External/**/*,*/obj/dotnet-new.IntegrationTests/**}]
+generated_code = true
+dotnet_diagnostic.IDE0005.severity = none
+dotnet_analyzer_diagnostic.severity = none
```

Comment:
please check if this is still needed, if not please merge with lines 18..21

Reply:
Yes, this is needed. External source in Wcwidth.Sources used by Cli violates the rules.

Reply:
I thought that `generatedCode` should prevent analyzers from being run. Can you please check if putting it on top of the file fixes it? If not, let's leave it as is.

Reply:
You're right. Thanks for explanation about `generatedCode`.